Jellyfin.Api/Models/StartupDtos/StartupConfigurationDto.cs

```diff
@@ -10,16 +8,16 @@ public class StartupConfigurationDto
         /// <summary>
         /// Gets or sets UI language culture.
         /// </summary>
-        public string UICulture { get; set; }
+        public string? UICulture { get; set; }
```

Comment:
I get an error if it's not nullable since it's not initialized. 
I can change it to `public string UICulture { get; set; } = null!;` if that is preferred

Reply:
System.Text.Json doesn't support it yet, but I'd like DTOs to be readonly struct. Not sure if @Bond-009 agrees or not.